src/ConnectedMode.UnitTests/Binding/BindingProcessImplTests.cs

```diff
@@ -352,6 +352,19 @@ public void EmitBindingCompleteMessage()
             testSubject.InternalState.BindingOperationSucceeded.Should().BeTrue($"Initial state of {nameof(BindingProcessImpl.InternalState.BindingOperationSucceeded)} should be true");
         }
 
+        [TestMethod]
+        public void SaveRuleConfiguration_CallsSolutionBindingOperation()
```

Comment:
Previously there were no tests for `SaveRuleConfiguration`.
It's a very simple method so it only needs this one test.